src/lib/settings.js

```diff
@@ -526,26 +529,156 @@ export function getDefaultSession() {
   return result;
 }
 
-function buildConfigurationMap201() {
-  const query = getDocumentQuery();
-  const list = configurationList201;
-  const result = {};
+class VariableConfiguration201 {
```

Comment:
I would inherit a class like `VariableConfiguration` in both `VariableConfiguration201` and `VariableConfiguration16` just to be clear that those classes share common methods (and must implement some of them)